### ISSUE TITLE
Error when cloning joomla repository

### ISSUE THREAD

**Issue by kstalega** [exakat/exakat#76]
HI, when I'm trying to clone Joomla! repository I got an error:
```php exakat.phar init -v -p joomla-3-8-3 -R https://github.com/joomla/joomla-cms.git```

This is the output:
```Initializing joomla-3-8-3 with https://github.com/joomla/joomla-cms.git
Check out with branch
An error prevented code initialization : 'Git reported an error and no code could be loaded : Remote branch master not found in upstream origin.'
.No code was loaded.
Done
```

What can I do to clone it? Thanks.

**Comment by kstalega** [comment]
Could you take a look on this?

**Comment by exakat** [comment]
I managed to reproduce this. Let me check what is wrong.

**Comment by exakat** [comment]
Exakat relied on master by default, while joomla uses staging. 
'master' default is being removed. Fixed in 1.5.3 (later today)

**Comment by kstalega** [comment]
It works now, thank you very much!
